VP-fitness-frontend/app/src/main/java/com/example/neil/vp_fitness_frontend/Tab1.java

```diff
@@ -0,0 +1,24 @@
+package com.example.neil.vp_fitness_frontend;
+
+import android.graphics.Typeface;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.avontell.fontutil.FontUtil;
+
+// Fragment class for Tab 1 of onboarding screen
+public class Tab1 extends Fragment {
```

Comment:
There are many other comments for these files that I have, but I will wait until the new infrastructure changes are made.